apps/dotcom/client/src/tla/components/dialogs/SubmitFeedbackDialog.tsx

```diff
@@ -140,47 +131,26 @@ function SignedInSubmitFeedbackDialog({ onClose }: { onClose(): void }) {
 					/>
 				</p>
 				<textarea
-					placeholder="Please provide as much detail as possible"
+					placeholder="Please provide as much detail as possible."
 					defaultValue={getFromLocalStorage(descriptionKey) ?? undefined}
 					onInput={(e) => {
 						setInLocalStorage(descriptionKey, e.currentTarget.value)
 					}}
-					style={{
-						border: '1px solid var(--tla-color-border)',
-						borderRadius: '6px',
-						minHeight: '100px',
-						padding: '6px 8px',
-						resize: 'vertical',
-					}}
+					className={styles.feedbackDialogTextArea}
 					ref={input}
 				/>
 			</TldrawUiDialogBody>
 			<TldrawUiDialogFooter className="tlui-dialog__footer__actions">
-				<div
-					style={{
-						display: 'flex',
-						flexDirection: 'column',
-						flex: 1,
-					}}
-				>
-					<div style={{ display: 'flex', flex: 1, padding: '0px 12px' }}>
-						<TlaFormCheckbox ref={checkBox} defaultChecked={false}>
-							<F defaultMessage="Allow tldraw to email me about this" />
```

Comment:
We need to keep this checkbox IMO, I don't wanna be emailing people to ask for more detail etc without their explicit consent.

Reply:
We can signpost on the form that we may email them to follow up, this isn't marketing so it's kosher